### ISSUE TITLE
Questions about val loss

### ISSUE THREAD

**Issue by LWYgut** [raymin0223/patch-mix_contrastive_learning#7]
Hi, @raymin0223. Thank you for opensource code !
When I reproduced your code, I found that the validation curve kept increasing, and the training curve started to increase around 30 epochs. Additionally, I followed your code and the score I obtained was only 58.46. Are these situations normal?
Thank you very much !
![four_loss_源码](https://github.com/raymin0223/patch-mix_contrastive_learning/assets/134259528/4315aab9-b8f0-4c89-b2df-43a81af6adf4)

**Comment by raymin0223** [comment]
Hi @LWYgut,

Thanks for your interests in our work!
As I have never plotted the loss curves, I'm not sure, but I believe that train loss was always decreased.
Also, val loss and accuracy were unstable and fluctuated a lot, and it seems like the issue might be due to the ICBHI dataset itself.

For your reproduced results, it looks weird if you use PatchMix CL loss.
Checking issue #3 might be helpful.

**Comment by LWYgut** [comment]
Okay, I understand. Thank you for your reply. Wishing you success in your work!
